vulcan_geometry.py

```diff
@@ -92,56 +86,33 @@ def addEightPack(instr, name: str, tube_type: str, upsidedown: bool = False):
     front    1 3 5 7
 
     This is similar to the incomplete function MantidGeometry.add_double_pack.
-    '''
-    type_element = le.SubElement(instr.root, 'type', name=name)
-    le.SubElement(type_element, 'properties')
-    component = le.SubElement(type_element, 'component', type=tube_type)
+
+    UPDATE: FROM PHASE-II ONWARDS UPSIDEDOWN=TRUE IS NO LONGER USED
```

Comment:
Would it be better to remove that parameter since it is never used?

Reply:
I would prefer keeping it in for a little bit longer (maybe till next phase) so that we are 100% sure we don't need it any more.